packages/analytics-react-native/ios/AppleContextProvider.swift

```diff
@@ -1,3 +1,5 @@
+import AdSupport
+import AppTrackingTransparency
```

Comment:
Thanks @falconandy.  I see why Swift has this externalized. To avoid the extra dependencies and warnings, I think we should remove the IDFA access from the core SDK and instead provide an example ReactNative plugin (TS) for [accessing IDFA](https://www.npmjs.com/package/@sparkfabrik/react-native-idfa-aaid) and setting it on the event payload.

Reply:
> remove the IDFA access from the core SDK and instead provide an example ReactNative plugin

Ready

Reply:
More info here. I think including idfa in the app will cause the app to get rejected from the app store. That's why we decided to  be customer driven. 
https://github.com/amplitude/Amplitude-iOS/blob/94b4dcfda9ed9edd6463d9a657bf30c9cf075f09/CHANGELOG.md?plain=1#L309